m4opt/missions/_ultrasat.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from regions import RectangleSkyRegion
+from synphot import Gaussian1D, SpectralElement
+
+from .. import skygrid
+from ..constraints import (
+    EarthLimbConstraint,
+    MoonSeparationConstraint,
+    SunSeparationConstraint,
+)
+from ..models import Detector
+from ..models.background import GalacticBackground, ZodiacalBackground
+from ..orbit import TLE
+from ..utils.dynamics import Slew
+from ._core import Mission
+
+ultrasat = Mission(
+    name="ultrasat",
+    fov=RectangleSkyRegion(
+        center=SkyCoord(0 * u.deg, 0 * u.deg), width=14.28 * u.deg, height=14.28 * u.deg
+    ),
+    constraints=[
+        EarthLimbConstraint(28 * u.deg),
+        SunSeparationConstraint(46 * u.deg),
+        MoonSeparationConstraint(23 * u.deg),
+        # GalacticLatitudeConstraint(10 * u.deg),
```

Comment:
Please remove commented-out code.

Reply:
@lpsinger, based on Michael's email and the ULTRASAT paper (page 5, https://arxiv.org/pdf/2304.14482), 
![ultrasat](https://github.com/user-attachments/assets/e2f9bec9-640f-4ecc-b72f-2eafbc44a9c5)
  
  I will update this to:
`   constraints=[
        EarthLimbConstraint(48 * u.deg),  
        SunSeparationConstraint(70 * u.deg),  
        MoonSeparationConstraint(35 * u.deg),
    ]

`